lib/error_store/interfaces/breadcrumbs.rb

```diff
@@ -0,0 +1,58 @@
+module ErrorStore::Interfaces
+  class Breadcrumbs < ErrorStore::BaseInterface
+    def self.display_name
+      'Breadcrumbs'
+    end
+
+    def type
+      :breadcrumbs
+    end
+
+    def sanitize_data(data)
+      values = []
+      data[:values].each do |value|
+        values << Breadcrumbs.new(@error).normalize_crumb(value)
```

Comment:
I don't think you need to do a new here of anything. Just call `normalize_crumb(value)`. The `@error` should be accessible in the method because it's defined by the BaseInterface class.